packages/compartment-mapper/test/dynamic-require.test.js

```diff
@@ -418,3 +418,45 @@ test('sync module transforms work without dynamic require support', async t => {
 
   t.is(transformCount, 2);
 });
+
+test('dynamic require of missing module falls through to importNowHook', async t => {
```

Comment:
could you point out which test is covering the situation where findRedirect finds the redirect?

Reply:
any test which contains a fixture which is absolute-requiring something is.  ... which is most of them.

Reply:
specifically, [this test](https://github.com/endojs/endo/blob/41c4494e5db51b00606491c49d31ff216a19f1df/packages/compartment-mapper/test/dynamic-require.test.js#L276-L350) covers the same sort of configuration as we encountered:

1. `hooked-app` statically requires package `hooked`
2. `hooked` dynamically requires package `dynamic` by absolute path
3. `dynamic` dynamically requires package `is-ok` by absolute path
4. `is-ok` statically requires package `is-not-ok`